### ISSUE TITLE
Important announcement for building for HoloLens or HMDs

### ISSUE THREAD

**Issue by StephenHodgson** [Microsoft/MixedRealityToolkit-Unity#1184]
If you're building for the HoloLens use the Vanilla **[Unity 2017.1.x](https://unity3d.com/unity/whats-new/unity-2017.1.2)** Release  and the 10.0.15063.0 SDK

If you're building for the WMR HMDs use the **[Unity 2017.2.0f3 MRTP](http://beta.unity3d.com/download/bf236913e22b/download.html)** and the latest 10.0.16299.0 SDK

**Comment by genereddick** [comment]
Is there an expected time frame / target release in which these two versions will merge into one?

**Comment by StephenHodgson** [comment]
They already are.

**Comment by genereddick** [comment]
Hmm, then I'm confused. The initial comment says to stay w 2017.1x if building for HoloLens and 2017.2 MRTP for WMR HMDs. Is it that the MRTK code base is now the same, and only the Unity version (and Windows SDK) that needs to vary?

**Comment by StephenHodgson** [comment]
That is correct.

**Comment by eluchsinger** [comment]
Are you planning on making everything work on 2017.2?

**Comment by StephenHodgson** [comment]
It's not us, it's unity we're waiting on.

**Comment by AnonyDev1** [comment]
Has this been fixed by Unity as of now, or do we still need to use 2017.1.x for HoloLens development?

**Comment by StephenHodgson** [comment]
long time ago.
